ayon_api/server_api.py

```diff
@@ -2322,7 +2323,8 @@ def get_dependency_packages(self):
         To download dependency package, use 'download_dependency_package'
         method and pass in 'filename'.
 
-        Example data structure:
+        Example data structure::
```

Comment:
Why double double colon?

Reply:
That is actually RST syntax to format everything as a code block following those double double colons:

In markdown, equivallent is:

---

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.

**Example data structure:**
    {
        "foo": "bar"
    }

---

With double double colons you'll get:

---
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.

**Example data structure:**
```
    {
        "foo": "bar"
    }
```

Reply:
For more, see  https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks